src/bench/bech32.cpp

```diff
@@ -13,9 +13,9 @@
 
 static void Bech32Encode(benchmark::Bench& bench)
 {
-    std::vector<uint8_t> v = ParseHex("c97f5a67ec381b760aeaf67573bc164845ff39a3bb26a1cee401ac67243b48db");
+    constexpr std::array<uint8_t, 32> v{ArrayFromHex("c97f5a67ec381b760aeaf67573bc164845ff39a3bb26a1cee401ac67243b48db")};
```

Comment:
btw, this is in the preprocessing phase of the benchmark, the performance doesn't matter here at all - only inside the `.run(` part, so I'd go with the cleanest code here, too.